### DISCUSSION TITLE
dynamic import with Suspense

### DISCUSSION THREAD

**Discussion by medihack** [blitz-js/blitz#2309] [Q&A]
Is it somehow possible to use [dynamic imports](https://blitzjs.com/docs/code-splitting#basic-usage) with [Suspense](https://reactjs.org/docs/concurrent-mode-suspense.html)? I know that `dynamic` supports a loading option, but I like to load multiple components in a list dynamically and show only one loading spinner for it. For this, `Suspense` would be a perfect fit. I tried to throw the returned promise of the `import` statement in several ways but without success. Maybe someone already has done this.

**Comment by flybayer** [comment]
Not currently. I did a quick search in the next.js repo and have seen any discussion on it either. But I think it would definitely make sense to add upstream in next.js
